grpc-ballerina/tests/17_bidirectional_chat_client.bal

```diff
@@ -57,8 +57,10 @@ public isolated function testBidiStreamingInChatClient() returns Error? {
     }
 
     var responseMsg = ep->receive();
-    if (responseMsg is error) {
+    if responseMsg is error {
         test:assertFail(msg = responseMsg.message());
+    } else if responseMsg is () {
+        test:assertFail(msg = "stream can't be without a response message");
```

Comment:
Isn't this the end of the stream?

Reply:
yes, but here we can expect a response message.

Reply:
the error message is incomplete. I will fix it